btrdb/endpoint.py

```diff
@@ -386,3 +386,25 @@ def sql_query(self, stmt, params: typing.List):
         for page in self.stub.SQLQuery(request):
             check_proto_stat(page.stat)
             yield page.SQLQueryRow
+
+    @error_handler
+    def subscribe(self, update_queue):
```

Comment:
Can we add type hinting for the `update_queue` parameter?

Reply:
Yeah, I think I will do a pass of type hinting + docstrings + tests